client/trino-jdbc/src/test/java/io/trino/jdbc/TestJdbcConnection.java

```diff
@@ -465,6 +467,21 @@ public void testConcurrentCancellationOnConnectionClose()
         testConcurrentCancellationOnConnectionClose(false);
     }
 
+    @Test
```

Comment:
Please move under `testSessionUser` method. The current place is located between `testConcurrentCancellationOnConnectionClose ()` and the helper method `testConcurrentCancellationOnConnectionClose(boolean autoCommit)`.

Reply:
Done